packages/legacy/core/App/components/misc/CredentialCard11.tsx

```diff
@@ -11,12 +13,57 @@ import Icon from 'react-native-vector-icons/MaterialIcons'
 import { useConfiguration } from '../../contexts/configuration'
 import { useTheme } from '../../contexts/theme'
 import { GenericFn } from '../../types/fn'
+import { NotificationStackParams, Screens } from '../../types/navigators'
 import { credentialTextColor, getCredentialIdentifiers, toImageSource } from '../../utils/credential'
 import { formatIfDate, getCredentialConnectionLabel, isDataUrl, pTypeToText } from '../../utils/helpers'
 import { testIdWithKey } from '../../utils/testable'
 
 import CardWatermark from './CardWatermark'
 
+interface CredentialActionFooterProps {
```

Comment:
Can this new component be moved to its own file? I feel like this file is getting a little unruly.

Reply:
Done